synapse/handlers/sso.py

```diff
@@ -72,18 +72,18 @@ async def get_sso_user_by_remote_user_id(
             The mxid of a previously seen user.
         """
         # Check if we already have a mapping for this user.
-        logger.info(
```

Comment:
Would it make sense to keep this, but at debug? Some of this is finicky to track down.

Reply:
After thinking about it, I suppose it can help in getting a timeline of things the server is doing down. Likewise for the other line. Thus it's probably worth keeping in.